app/scripts/check_requirements

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env php
+<?php
+
+// Script should not be called from the Web
+if (php_sapi_name() != 'cli') {
+    die('Nope');
+}
+
+/**
+ * Colorize the shell output
+ *
+ * @param string $text Text to display.
+ * @param string $color Color of the text: green, red or nothing.
+ *
+ * @return string
+ */
+function colorizeOutput($text, $color) {
+    switch ($color) {
+        case 'green':
+            // White on green
+            $output = "\033[1;37m\033[42m{$text}";
+            break;
+        case 'red':
+            // White on red
+            $output = "\033[1;37m\033[41m{$text}";
+            break;
+        default:
+            $output = $text;
+            break;
+    }
+    $output .= "\033[0m";
+    
+    return $output;
+}
+
+/**
+ * Check current configuration.
+ * 
+ * @param bool   $condition The condition to check.
+ * @param string $message   The message to display.
+ * @param string $help      Help about the message.
+ * @param string $suggested_fix A message that helps fixing.
+ *
+ * @return void
+ */
+function check($condition, $message, $help = '', $suggested_fix = '') {
+    $result = $condition ? colorizeOutput('OK', 'green') : colorizeOutput('FAILED', 'red');
+    $text = "* {$message}: {$result}\n";
+
+    if (! $condition) {
+        if(! empty($help)) { 
+            $text .=  "  {$help}\n";
+        }
+        if(! empty($suggested_fix)) {
+            $text .=  "  {$suggested_fix}\n";
+        }
+    }
+    
+    echo $text;
+}
+
+/**
+ * Check if the directory is empty
+ *
+ * @param string $dir The path.
+ *
+ * @return bool
+ */
+function isEmptyDir($dir) {
+    if(! is_dir($dir)) {
+        return false;
```

Comment:
I would return true and change the failure message to "TMX folder is empty or missing".